domain/oldProgrammes.js

```diff
@@ -0,0 +1,5 @@
+export const OLD_PROGRAMME_TYPES = ['1993PRG']
+
+export function isOldProgramme(programType) {
+  return OLD_PROGRAMME_TYPES.includes(programType)
+}
```

Comment:
When I ran locally, it expected CommonJS syntax (and not ES module mode). As all other domain files have this, I think we should have it in this file as well, like:

```suggestion
const OLD_PROGRAMME_TYPES = ['1993PRG']

function isOldProgramme(programType) {
  return OLD_PROGRAMME_TYPES.includes(programType)
}

module.exports = {
  OLD_PROGRAMME_TYPES,
  isOldProgramme,
}
```